panel/layout.py

```diff
@@ -107,6 +111,7 @@ def _cleanup(self, root):
         super(Panel, self)._cleanup(root)
         for p in self.objects:
             p._cleanup(root)
+        silence(EMPTY_LAYOUT, False)
```

Comment:
Could you explain this? This seems weird, I can imagine creating a panel, displaying it, deleting the cell or replacing the panel in the plot which triggers cleanup and then a subsequent display of the panel would again have the warning enabled again.

Reply:
I left that in by mistake. I was trying to make it cleanup after itself. But this isn't the right way.